src/forms/TopComponent/Utils/TitleWrapper.tsx

```diff
@@ -6,10 +6,14 @@ import styled from "styled-components";
 
 // Utils
 import theme, { BREAKPOINTS, Views } from "@root/theme/theme";
+import Button from "@root/components/Button";
+import ChevronLeftIcon from "@mui/icons-material/ChevronLeft";
 
 const BIG_SCREEN_BREAKPOINT = BREAKPOINTS.topComponent.bigScreenView + "px";
 
 export const FormTitle = styled.span`
+	display: flex;

```

Comment:
Warning: Most of this will get overwritten when getting merged with the other ticket. No action for now until the merge conflicts appear.